tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -579,14 +579,39 @@ internal static class AppSec
 
             internal static string Track(string eventName) => $"{Events}{eventName}.track";
 
-            internal static class EventsUsersLogin
+            internal static class EventsUsers
             {
-                internal const string Success = AppSec.Events + "users.login.success.";
-                internal const string Failure = AppSec.Events + "users.login.failure.";
-                internal const string SuccessTrack = Success + "track";
-                internal const string FailureTrack = Failure + "track";
-                internal const string FailureUserId = Failure + "usr.id";
-                internal const string FailureUserExists = Failure + "usr.exists";
+                internal const string EventsUsersRoot = Events + "users";
+
+                internal static class LoginEvent
+                {
+                    private const string Root = EventsUsersRoot + ".login";
+                    internal const string Success = Root + ".success";
+                    internal const string SuccessSdkSource = $"_dd.{Success}.sdk";
```

Comment:
These `sdk` tags never seem to appear in the snapshots, should they? 🤔

Reply:
yes, we dont have integration tests for the sdk, but unit tests like `EventTrackingSdkTests` where we assess"
```
        Assert.Equal("true", traceContext.Tags.GetTag(Tags.AppSec.EventsUsers.LoginEvent.SuccessSdkSource));
```